src/seedu/addressbook/data/person/Person.java

```diff
@@ -24,7 +26,8 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.phone = phone;
         this.email = email;
         this.address = address;
-        this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list        
+        sequenceNumber = ++nextSequenceNumber;
```

Comment:
Although this works, it's more intuitive to initialize `nextSequenceNumber` to `1` and increment it *after* it has been assigned. As the variable name has the word `next`, it should hold the sequence number for the next Person object, not the most recent one.